operations/helm/tests/test-enterprise-configmap-values-generated/mimir-distributed/templates/mimir-config.yaml

```diff
@@ -0,0 +1,145 @@
+---
+# Source: mimir-distributed/templates/mimir-config.yaml
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: test-enterprise-configmap-values-mimir-config
+  labels:
+    app.kubernetes.io/name: mimir
+    app.kubernetes.io/instance: test-enterprise-configmap-values
+    app.kubernetes.io/version: "2.1.0"
+    app.kubernetes.io/managed-by: Helm
+data:
+  mimir.yaml: |
+    
+    
+    
+    limits: {}
+    
+    activity_tracker:
+      filepath: /data/metrics-activity.log
+    
+    alertmanager:
+      data_dir: '/data'
+      enable_api: true
+      external_url: '/alertmanager'
+    alertmanager_storage:
+      backend: s3
+      s3:
+        endpoint: test-enterprise-configmap-values-minio.citestns.svc:9000
+        bucket_name: mimir-ruler
+        access_key_id: grafana-mimir
+        secret_access_key: supersecret
```

Comment:
It would be neat to pull this from the `test-enterprise-configmap-values-minio` secret, but that could be done in a future PR with structured config to avoid copying the whole config string

Reply:
way ahead of you :) I started to work on a PR that pulls together all the config improvement PRs and tests what we'd do in real life